spec/models/business_finance_support_scheme_spec.rb

```diff
@@ -1,11 +0,0 @@
-require "spec_helper"
-require "models/valid_against_schema"
-
-RSpec.describe BusinessFinanceSupportScheme do
-  let(:payload) { FactoryBot.create(:business_finance_support_scheme) }
-  include_examples "it saves payloads that are valid against the 'specialist_document' schema"
-
-  it "is exportable" do
```

Comment:
I think we need to keep these tests, as we're not testing `self.exportable?` anywhere else.

(We could remove these tests if we write a test that an `exportable` boolean in the schema can be surfaced by `<Finder>.exportable?` but it doesn't look like that property exists in the schema yet  - it's hardcoded in the model).

Reply:
Ah yep, you're right. Think I removed by mistake!